### PR TITLE
Add `DeleteFriend` and `ConferenceInfo`

### PR DESCRIPTION
**Pull request by suhr**

### CODE REVIEW

File: lib/protocol.schema.json [hunk 1]
```diff
@@ -700,6 +712,27 @@
...
+                "public_key": { "type": "string" },
+                "name": { "type": "string" }
+            },
+            "required": [ "number", "public_key", "name" ]
+        },
+        "ConferenceInfo": {
+            "type": "object",
+            "properties": {
+                "number": { "type": "number" },
+                "kind": { "enum": ["Text", "Av"] },
```
Comment (kpp) [inline lib/protocol.schema.json:729]: $ref?

File: src/lib.rs [hunk 2]
```diff
@@ -106,12 +108,13 @@ pub enum Response {
...
 
     Conference { conference: u32 },
+    ConferencePeerList { peers: Vec<PeerInfo> },
     ConferencePeerCount { count: u32 },
     ConferencePeerName { name: String },
     ConferencePeerPublicKey { public_key: String },
     IsOwnPeerNumber { is_own: bool, },
     ConferenceTitle { title: String, },
-    ChatList { list: Vec<u32> },
+    ChatList { conferences: Vec<ConferenceInfo> },
```
Comment (kpp) [inline src/lib.rs]: ConferencePeerList vs ChatList? Why not ConferenceList?

Comment (suhr) [reply src/lib.rs]: Name from the Tox API. I probably should change it.

Comment (kpp) [reply src/lib.rs]: > I probably should change it.

I think so.

File: lib/protocol.schema.json (hunk 1 above)
Comment (suhr) [reply lib/protocol.schema.json:729]: Enums are unwrapped everywhere. Maybe I should move them in separate definitions, but this I'll do it later (in a separate PR)

**Review by kpp — APPROVED** [review]

**Pull request merged**